packages/bitcore-wallet-service/lib/blockchainexplorers/v8.js

```diff
@@ -15,192 +15,196 @@ var Bitcore_ = {
   btc: Bitcore,
   bch: require('bitcore-lib-cash')
 };
+var config = require('../../config');
 
 var Constants = Common.Constants,
   Defaults = Common.Defaults,
   Utils = Common.Utils;
 
-
 function v8network(bwsNetwork) {
   if (bwsNetwork == 'livenet') return 'mainnet';
+  if (
+    bwsNetwork == 'testnet' &&
+    config.blockchainExplorerOpts.btc.testnet.regtestEnabled
+  )
```

Comment:
Conditional if network is testnet and regtestEnabled is true, change testnet to regtest